assets/js/components/SupportLink.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_SITE } from '../googlesitekit/datastore/site/constants';
+import Link from './Link';
+const { useSelect } = Data;
+
+export default function SupportLink( props ) {
+	const { path, query, hash, ...otherProps } = props;
+
+	const supportURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getGoogleSupportURL( { path, query, hash } )
+	);
+
+	return <Link { ...otherProps } href={ supportURL } />;
+}
+
+SupportLink.propTypes = {
+	path: PropTypes.string.isRequired,
+	query: PropTypes.object,
+	hash: PropTypes.hash,
```

Comment:
This is raising an error in tests – I think it should be a string, correct?

```
Warning: Failed prop type: SupportLink: prop type `hash` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.
          in SupportLink
```

Reply:
Ah.. good catch. Thanks, Evan. Should be fixed now.